src/quic/node_quic_crypto.cc

```diff
@@ -80,24 +81,24 @@ bool DeriveTokenKey(
           &ctx.aead,
           &ctx.md,
           secret,
-          NGTCP2_CRYPTO_TOKEN_SECRETLEN));
+          kCryptoTokenSecretlen));
 }
 
 void GenerateRandData(uint8_t* buf, size_t len) {
-  std::array<uint8_t, 16> rand;
-  std::array<uint8_t, 32> md;
+  uint8_t rand[16];
+  uint8_t md[32];
 
   const EVP_MD* meth = EVP_sha256();
   unsigned int mdlen = EVP_MD_size(meth);
   DeleteFnPtr<EVP_MD_CTX, EVP_MD_CTX_free> ctx;
   ctx.reset(EVP_MD_CTX_new());
-  CHECK(ctx);
+  DCHECK(ctx);
```

Comment:
Ditto here and elsewhere – if this condition is ever false, there’s a crash that’s a lot harder to debug without the `CHECK` message. `CHECK`s are ultimately very cheap (branch prediction is basically always going to do the right things for them), especially if they only perform single-word comparisons; I’d use `DCHECK`s only when there’s real performance overhead associated with the condition itself or in really hot paths (after benchmarking).